docs/csharp/getting-started/with-visual-studio-code.md

```diff
@@ -13,16 +13,45 @@ ms.assetid: 76c23597-4cf9-467e-8a47-0c3703ce37e7
 
```

Comment:
change author to your GH alias and ms.date to today's date. It'd be good to review the description too but we'll review that later as well.